srcpkgs/backblaze-b2/patches/tqdm_ver.patch

```diff
@@ -0,0 +1,10 @@
+Upstream builds with the newer version of tqdm, but upstream hasn't updated their requirements.txt
+
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -7,4 +7,4 @@
+ phx-class-registry~=4.0
+ rst2ansi==0.1.5
+ tabulate==0.9.0
+-tqdm~=4.65.0
++tqdm~=4.66.0
```

Comment:
```suggestion
+tqdm>=4.65.0
```
Weaken the restriction to avoid version issues in the future.

Reply:
Will get that pushed momentarily